.github/labeler.yml

```diff
@@ -0,0 +1,55 @@
+ci/run-unit/@cdktf/cli-core:
+  # The workflow used
+  - ".github/workflows/unit.yml"
+  # All packages under test
+  - "packages/@cdktf/cli-core/**"
+  # All scripts we use
+  - "tools/align-version.sh"
```

Comment:
I'm wondering whether we could simplify this config by adding `ci/run-unit` with those two repeated files (that are common for all individual unit labels currently). It would then effectively run all unit tests if we change those files while not spamming the PR with all individual unit test labels 🤔

Reply:
Sounds good, will do it in a follow-up though